UnitsNet.Tests/CustomCode/EnergyTests.cs

```diff
@@ -55,7 +55,7 @@ public class EnergyTests : EnergyTestsBase
         protected override double WattHoursInOneJoule => 0.000277777778;
 
         [Fact]
-        public void EnergyCreatedInSISystem()
+        public void Constructor_UnitSystem_SIUsesSIUnits()
```

Comment:
I know  there is no consistent pattern in our test suite, but if you are dividing up the name like this I would expect it to read as {what is tested}\_{what are the inputs}\_{what is the result}. At least that is a common naming scheme I've seen a lot.

So the middle part should then say this is a SI unit system and the final part say something about SI units being the result - or maybe just spell it out directly that Joule is the unit.

Something along the lines of `Constructor_UnitSystemSI_AssignsSIUnit` or `Constructor_GivenUnitSystemSI_AssignsJouleUnit`. Same for test methods below.

Reply:
Done!